### ISSUE TITLE
defaultSymbolDictName specified in a component does not take hold

### ISSUE THREAD

**Issue by dalehenrich** [GemTalk/Rowan#527]
RwRowanSample4Test>>test_projectUrl_issue_463 needs the defaultSymbolDictName: message  (line 14)to avoid loading project into UserGlobals:
```smalltalk
test_projectUrl_issue_463
	| specUrlString rowanProject testProjectHome projectName project projectDef |
	projectName := 'RowanSample4'.
	(Rowan image loadedProjectNamed: projectName ifAbsent: [  ])
		ifNotNil: [ :prj | Rowan image _removeLoadedProject: prj ].
	rowanProject := Rowan image _projectForNonTestProject: 'Rowan'.
	specUrlString := self _rowanSample4LoadSpecificationUrl.
	testProjectHome := (rowanProject repositoryRootPath
		, '/test/testRepositories/repos/') asFileReference.
	(testProjectHome / projectName) exists
		ifTrue: [ (testProjectHome / projectName) deleteAll ].
	projectDef := (RwComponentProjectDefinition newForUrl: specUrlString)
		projectHome: testProjectHome;
"--->"	defaultSymbolDictName: self _symbolDictionaryName1;
		yourself.
	self
		handleInformDuring: [ projectDef clone ]
		interactionBlock: [ :inform | self assert: false description: 'unexpected inform' ].
	self
		assert: projectDef projectUrl = 'https://github.com/dalehenrich/RowanSample4'.
	projectDef load.
	project := RwProject newNamed: projectName.
	self assert: project projectUrl = projectDef projectUrl```
Look at commit `fb3fdc5` in RowanSample4 ...

Ending up with a RowanSample4 project installed in UserGlobals gums up the batch testing works, so I've got a failing test with this bug number as a placeholder

**Comment by dalehenrich** [comment]
The root cause of this issue is related to the fact that the loadedProjectInfo is created but not immediately applied to the RwCompoentProjectDefinition ... this is a carryover from v1.2 ... in v2.0, we do the immediate application of loaded projectInfo ... 

in RwRowanSample4Test>>testIssue230, the loaded info is created during the `clone` of the projectDefinition and the clone does involve a `read` of the project from disk, which produces the loadedProjectInfo, but does not apply it ...

There are mechanisms that currently explicitly look for and apply loadedProjectInfo and those boys need to be snuffed out ...  

Issue #500 explicitly calls out for doing this automatically and now is as good a time as any ...

**Comment by dalehenrich** [comment]
After a discussion with Kurt, I've decided that we need to go back to having a "first class" load specification that is used to specify all of the options for a project load ... the load spec is preserved intact for a loaded project ... the load spec is edited if you want to change what is loaded ... 

I'll start with a fresh class for the load spec and start (hopefully) the final round of tweaking the load api..

**Comment by dalehenrich** [comment]
with the fix for #530 (which came up during the work on fixing this bug), it is time to rewrite RowanSample4Test>>test_projectUrl_issue_527 against RowanSample9. RowanSample9Test>>testIssue_527 seems to cover this issue, but I want to double check before closing this issue against V2 ...

**Comment by dalehenrich** [comment]
defaultSymbolDictName handling has been significantly improved with the introduction of RwProjectLoadSpecificationV2
